QuantConnect.BybitBrokerage.Tests/BybitFuturesBrokerageHistoryProviderTests.cs

```diff
@@ -59,137 +36,23 @@ private static TestCaseData[] ValidHistory
-                historyProvider.SetBrokerage(_brokerage);
-                historyProvider.Initialize(new HistoryProviderInitializeParameters(null, null, null,
-                    null, null, null, null,
-                    false, new DataPermissionManager(), null));
-
-                var marketHoursDatabase = MarketHoursDatabase.FromDataFolder();
-
-                var now = DateTime.UtcNow.AddDays(-1);
-                var requests = new[]
-                {
-                    new HistoryRequest(now.Add(-period),
-                        now,
-                        resolution == Resolution.Tick ? typeof(Tick) : typeof(TradeBar),
-                        symbol,
-                        resolution,
-                        marketHoursDatabase.GetExchangeHours(symbol.ID.Market, symbol, symbol.SecurityType),
-                        marketHoursDatabase.GetDataTimeZone(symbol.ID.Market, symbol, symbol.SecurityType),
-                        resolution,
-                        false,
-                        false,
-                        DataNormalizationMode.Adjusted,
-                        tickType)
-                };
-
-                var historyArray = historyProvider.GetHistory(requests, TimeZones.Utc).ToArray();
-                foreach (var slice in historyArray)
-                {
-                    if (resolution == Resolution.Tick)
-                    {
-                        foreach (var tick in slice.Ticks[symbol])
-                        {
-                            Log.Debug($"{tick}");
-                        }
-                    }
-                    else if (slice.QuoteBars.TryGetValue(symbol, out var quoteBar))
-                    {
-                        Log.Debug($"{quoteBar}");
-                    }
-                    else if (slice.Bars.TryGetValue(symbol, out var tradeBar))
-                    {
-                        Log.Debug($"{tradeBar}");
-                    }
-                }
-
-                Assert.Greater(historyProvider.DataPointCount, 0);
-
-                if (historyProvider.DataPointCount > 0)
-                {
-                    // Ordered by time
-                    Assert.That(historyArray, Is.Ordered.By("Time"));
-
-                    // No repeating bars
-                    var timesArray = historyArray.Select(x => x.Time).ToArray();
-                    Assert.AreEqual(timesArray.Length, timesArray.Distinct().Count());
-                }
-
-                Log.Trace("Data points retrieved: " + historyProvider.DataPointCount);
-            };
-
-            if (throwsException)
-            {
-                Assert.Throws<ArgumentException>(test);
-            }
-            else
-            {
-                Assert.DoesNotThrow(test);
-            }
+            base.GetsHistory(symbol, resolution, period, tickType, throwsException);
         }
 
-        protected virtual Brokerage CreateBrokerage()
+        [Ignore("Same as base")]
```

Comment:
Maybe the message can be improved to make more explicit that we don't want to run these method as a test since it's the same cases as the base one. Also could just leave the method empty to make it more explicit.